src/odemis/acq/fastem_conf.py

```diff
@@ -143,3 +143,22 @@ def configure_scanner(scanner, mode):
                             scanner.rotation.value)
     else:  # code should never be reached
         raise ValueError("Invalid mode %s." % mode)
+
+
+def configure_detector(detector, roc):
+    """
+    Configure the detector by setting the calibrated parameter as stored for
+    the provided region of calibration (ROC). If calibrated parameters are not
+    available, the current settings on the detector will stay as is.
+    :param detector: (technolution.MPPC) The detector to be configured.
+    :param roc: (FASTEMROC) The region of calibration as selected on the scintillator,
+                which stores the calibrated settings if calibration was performed.
+    """
+    if "cellDarkOffset" in roc.value.parameters:
+        detector.cellDarkOffset.value = roc.value.parameters["cellDarkOffset"]
+    else:
+        logging.warning("Region of calibration doesn't have dark offset parameters.")
```

Comment:
Not necessarily. It is two-fold. If it cannot find any parameters it indeed it will use the currently set values on the VAs. BUT, this method is called with two different ROCs. The first one will only have dark/gain parameters, so it will never have cell translation parameters and will always trigger the warning. However, it will not use the VA settings for cell translation parameters as the same function is also called with the ROC that actually contains the cell translation parameters. 
So I cannot add that the VA values are used. We can maybe change how the method is used by the 2 ROCs. But I don't have a good idea yet.

Reply:
Will change input arg to list of ROCs in pr for calibration step 3.